docs/docs/Data Validation/Known Issues.md

```diff
@@ -9,154 +9,187 @@ We maintain a [list of these known issues on GitHub](https://github.com/singular
-305             508                                 SUB                WND                                         coal                        wind
-333             552                                 DFO                WAT                                    petroleum                       hydro
-336             557                                  JF                WAT                                    petroleum                       hydro
+2024-04-03 12:47:48,742 [WARNING] oge.oge.validation:209 
+     plant_id_eia plant_primary_fuel_from_capacity_mw plant_primary_fuel plant_primary_fuel_from_capacity_mw_category plant_primary_fuel_category ba_code
+0              63                                 DFO                WAT                                    petroleum                       hydro    AKMS
+1              91                                 DFO                WAT                                    petroleum                       hydro    AKMS
+2             160                                  NG                SUB                                  natural_gas                        coal    WALC
+3             165                                  NG                SUB                                  natural_gas                        coal    SWPP
+4             557                                  JF                WAT                                    petroleum                       hydro    ISNE
+5             667                                 BIT                 NG                                         coal                 natural_gas     JEA
+6             899                                  NG                WAT                                  natural_gas                       hydro    MISO
+7             944                                  NG                WND                                  natural_gas                        wind    MISO
+8             955                                 DFO                WAT                                    petroleum                       hydro    MISO
+9             971                                 DFO                 NG                                    petroleum                 natural_gas    MISO
 [...]
 ```
 
 ### Missing NOx and SO2 emission factors for Fuel Cells
 Our validation check identifies that there are missing NOx and SO2 emission factors for plants with fuel cell prime movers. This is a known and expected issue - to our knowledge emission factors for fuel cells have not yet been published.
 
+
+### Global extreme are detected in CEMS timeseries
+The CEMS gross generation, fuel consumed and CO2 emission timeseries are screened for global extremes in a validation check. The check reports the number of hours where a value exceeds the median value by at least 10 times together with the mean deviation from the median of all the global extremes.
```

Comment:
Suggested edit:
We identify where gross generation, fuel consumption, and CO2 emission timeseries in the reported CEMS data may be anomalous. The check reports the number of observations where the hourly value exceeds the median value for the unit by a factor of 10 or more, as well as the mean deviation of these extreme values from the median.

Reply:
Done